pallets/committee-management/src/migration.rs

```diff
@@ -0,0 +1,109 @@
+use frame_support::{
+    log::info,
+    migration::move_storage_from_pallet,
+    pallet_prelude::{Get, PalletInfoAccess, StorageVersion},
+    storage::generator::StorageValue,
+    traits::OnRuntimeUpgrade,
+    weights::Weight,
+    StoragePrefixedMap,
+};
+#[cfg(feature = "try-runtime")]
+use {
+    frame_support::{ensure, traits::STORAGE_VERSION_STORAGE_KEY_POSTFIX},
+    pallets_support::ensure_storage_version,
+    sp_io::hashing::twox_128,
+    sp_std::vec::Vec,
+};
+
+use crate::{
+    BanConfig, Banned, Config, Pallet, SessionValidatorBlockCount,
+    UnderperformedValidatorSessionCount, ValidatorEraTotalReward,
+};
+
+const LOG_TARGET: &str = "pallet-committee-management";
+const OLD_PREFIX: &str = "Elections";
+
+/// migrate prefixes from Elections to this pallet.
+pub struct PrefixMigration<T>(sp_std::marker::PhantomData<T>);
+
+impl<T: Config> OnRuntimeUpgrade for PrefixMigration<T> {
+    fn on_runtime_upgrade() -> Weight {
+        if StorageVersion::get::<Pallet<T>>() != StorageVersion::new(0) {
```

Comment:
As we add this pallet, we should set the version explicitly, so it's actually stored and not only return default value of 0.

Reply:
done